wgpu-hal/src/gles/mod.rs

```diff
@@ -240,28 +240,40 @@ enum TextureInner {
         raw: glow::Texture,
         target: BindTarget,
     },
+    #[cfg(all(target_arch = "wasm32", not(target_os = "emscripten")))]
+    ExternalFramebuffer {
+        inner: web_sys::WebGlFramebuffer,
+    },
 }
 
+// SAFE: WASM doesn't have threads
```

Comment:
WASM can have threads.

Reply:
Well, technically yes but we're ignoring that for now :P. There are some other comments in this file with the same assumption and when threads in wasm are more readily used, we'll have to change things.